core/domain/exp_services.py

```diff
@@ -123,6 +136,15 @@ def get_exploration_from_model(exploration_model, run_conversion=True):
     IMPORTANT NOTE TO DEVELOPERS: In general, run_conversion should never be
     False. This option is only used for testing that the states schema version
     migration works correctly, and it should never be changed otherwise.
+
+    Args: 
+        exploration_model: Exploration domain object of exploration model.
```

Comment:
No, this is the storage model.

    exploration_model: An exploration storage model.
    run_conversion: bool. If True, updates the exploration to the latest states_schema_version if necessary.